.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+# Docs: https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: ".github:"
+
+  # Maintain dependencies for Go
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
@conduitio-labs/conduit-core maybe we can change this to weekly or biweekly?

Reply:
Whoops, I missed this. I don't have a strong opinion, I don't see an issue with the current interval though, since I think we have the same in all connectors.